integration-tests/src/test/java/oracle/kubernetes/operator/ITStickySession.java

```diff
@@ -0,0 +1,302 @@
+   *
+   * @throws Exception
+   */
+  @Test
+  public void testDiffSessionsNoSharing() throws Exception {
+    Assume.assumeFalse(QUICKTEST);
+    String testMethodName = new Object() {}.getClass().getEnclosingMethod().getName();
+    logTestBegin(testMethodName);
+
+    Map<String, Object> domainMap = domain.getDomainMap();
+    String domainNS = domainMap.get("namespace").toString();
+    String domainUid = domain.getDomainUid();
+
+    int counterNum = 4;
+    String webServiceSetUrl = testAppPath + "?setCounter=" + counterNum;
+    String webServiceGetUrl = testAppPath + "?getCounter";
+
+    String sessionIDAttr = "sessionid";
+    String countAttr = "count";
+
+    // Client1 sends a HTTP request to set a count number
+    ExecResult result = getHTTPResponse(webServiceSetUrl);
+
+    // Retrieve the session id from HTTP response
+    String serverIDClient1 = getHttpResponseAttribute(result.stdout(), sessionIDAttr);
+    Assume.assumeNotNull(serverIDClient1);
+
+    logger.info(
+        "Client1 created a connection with HTTP session id <"
+            + serverIDClient1
+            + "> and set a count number to <"
+            + counterNum
+            + ">");
+
+    // Client2 sends a HTTP request to get a count number
+    result = getHTTPResponse(webServiceGetUrl);
+
+    // Retrieve the session id and count number from HTTP response
+    String serverIDClient2 = getHttpResponseAttribute(result.stdout(), sessionIDAttr);
+    String count = getHttpResponseAttribute(result.stdout(), countAttr);
+
+    Assume.assumeNotNull(serverIDClient2);
+    Assume.assumeNotNull(count);
+
+    logger.info(
+        "Client2 created a connection with HTTP session id <"
+            + serverIDClient2
+            + "> and retrieved a count number <"
+            + count
+            + ">");
+
+    // Verify that each client session has its own session ID
+    Assume.assumeFalse("HTTP session should NOT be same!", serverIDClient1.equals(serverIDClient2));
+
+    // Verify that count number retrieved from session state is not shared between two clients
+    Assume.assumeTrue(
+        "Count number <" + counterNum + "> set by client1 should be invisible to client2",
+        count.equals("0"));
+
+    logger.info("SUCCESS - " + testMethodName);
+  }
+
+  private String getHttpResponseAttribute(String httpResponseString, String attribute)
+      throws Exception {
+
+    String attrPatn = httpAttrMap.get(attribute);
+
+    Assume.assumeNotNull(attrPatn);
+
+    String httpAttribute = null;
+
+    Pattern pattern = Pattern.compile(attrPatn);
+    Matcher matcher = pattern.matcher(httpResponseString);
+
+    if (matcher.find()) {
+      httpAttribute = matcher.group(2);
+    }
+
+    return httpAttribute;
+  }
+
+  private ExecResult getHTTPResponse(String webServiceURL, String... args) throws Exception {
+    String headerOption = (args.length == 0) ? "" : args[0] + httpHeaderFile;
+
```

Comment:
remove new line here

Reply:
I feel that a newline is needed here to make it easy to read and looks clearer.